.github/workflows/python_push_master.yml

```diff
@@ -0,0 +1,107 @@
+name: Python - Master Push
+
+# runs on master and tag (aka release) pushes to build all current platforms & minor python versions
+# publishes to https://pypi.org/project/pyvalhalla/ for tag pushes
+# publishes to https://pypi.org/project/pyvalhalla-git/ for master pushes
+
+on:
+  push:
```

Comment:
we'll also run this on `tag` pushes, but not on PRs. we run the full python test suite on each PR for linux & OSX already. for the packages bindings we run a much simpler test, only making sure we can import the `Actor`.